### PR TITLE
add more IP logging

### PR DESCRIPTION
**Pull request by cloudedge**
help troubleshoot case where there's no IP that answers

### CODE REVIEW

File: rails/app/models/secure_shell_hammer.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ def actions
   end
 
   def run(cmd)
```
Comment (VictorLowther) [inline rails/app/models/secure_shell_hammer.rb]: Doing it this way runs the address detect loop twice.

Comment (cloudedge) [inline rails/app/models/secure_shell_hammer.rb]: oy!  good catch

Comment (cloudedge) [inline rails/app/models/secure_shell_hammer.rb]: but only on the error case

Comment (VictorLowther) [inline rails/app/models/secure_shell_hammer.rb]: Nope, it only runs node.address once in the error case. In the normal case it will run twice.

Comment (cloudedge) [inline rails/app/models/secure_shell_hammer.rb]: Addressed.

**Comment by galthaus** [comment]
Including to try and help find some field issues.

**Pull request merged**